src/factorio_randovania_mod/lua_src/prototypes/tech.lua

```diff
@@ -6,22 +9,28 @@ local function make_tech(param)
         param.prerequisites = { param.cost_reference }
         param.cost_reference = nil
     end
+    param.randovania_custom_tech = true
+    assert(param.unit)
     return param
 end
 
+local impossible_lab = table.deepcopy(data.raw["lab"]["lab"])
+impossible_lab.name = "impossible-lab"
+impossible_lab.inputs = { "impossible-science-pack" }
+
```

Comment:
add a comment explaining why this is necessary since it isn't exactly obvious imo